ext/ffi/lib.rs

```diff
@@ -825,8 +737,17 @@ fn make_sync_fn<'s>(
   )
   .data(v8::External::new(scope, sym as *mut Symbol as *mut _).into());
 
-  let func = if let Some(fast_ffi_templ) = fast_ffi_templ {
-    builder.build_fast(scope, &fast_ffi_templ, None)
+  let mut fast_call_alloc = None;
+
+  let func = if fast_call::is_compatible(sym) {
+    let trampoline = fast_call::compile_trampoline(sym);
```

Comment:
Huh, I would have liked to have suggested making `compile_trampoline` return an `Option`, but I guess that you can use `compile_trampoline` without `is_compatible`, right?